data/widgets/timeline_conf.xml

```diff
@@ -2,6 +2,27 @@
 <!-- Copyright (C) 2014, 2015 by David Capello -->
 <gui>
 <vbox id="timeline_conf">
+  <separator cell_hspan="2" text="Cel Preview:" left="true" horizontal="true" />
+  <grid columns="2">
+    <hbox cell_hspan="2">
+      <check id="cel_show_thumb" text="Show thumbnails" cell_hspan="2" />
+      <check id="cel_show_overlay" text="Show on mouse hover" cell_hspan="2" />
+    </hbox>
+
+    <hbox cell_hspan="2">
+      <label text="Thumb quality:" />
+      <radio group="3" text="Nearest" id="cel_quality_nearest" />
+      <radio group="3" text="Bilinear" id="cel_quality_bilinear" />
+      <radio group="3" text="Rotsprite" id="cel_quality_rotsprite" />

```

Comment:
I think this "quality" parameter might be displayed as a combo-box as in the resize dialog ([see this](https://github.com/aseprite/aseprite/blob/master/data/widgets/sprite_size.xml#L40) and [this](https://github.com/aseprite/aseprite/blob/master/src/app/commands/cmd_sprite_size.cpp#L171)).